examples/plot_omni.py

```diff
@@ -17,11 +17,11 @@
 omni_data = omni.low(starttime, endtime)
 
 fig, axs = plt.subplots(3, 1, sharex=True)
-axs[0].plot(omni_data.data['Bx GSE, GSM'])
-axs[1].plot(omni_data.data['By GSE'])
-axs[1].plot(omni_data.data['Bz GSE'])
-axs[2].plot(omni_data.data['By GSM'])
-axs[2].plot(omni_data.data['Bz GSM'])
+axs[0].plot(omni_data['Bx GSE, GSM'].data)
+axs[1].plot(omni_data['By GSE'].data)
+axs[1].plot(omni_data['Bz GSE'].data)
+axs[2].plot(omni_data['By GSM'].data)
+axs[2].plot(omni_data['Bz GSM'].data)
```

Comment:
These lines are causing the CircleCI error - `omni_data` seems to still be a `GenericTimeSeries`.